app/models/project.rb

```diff
@@ -215,6 +215,10 @@ def environment_variables_with_scope
     )
   end
 
+  def external_environment_variable_groups
```

Comment:
- same here, should be done in the plugin
 - name is plural, but returns a single one ... seems fishy

Reply:
already exists as external_environment_variable_groups